src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -662,10 +688,33 @@ protected function getConfig()
         }
         $settings['theme_url'] = $theme;
 
+        $this->initImageSizePresets($settings);
+
         // Send back
         return $settings;
     }
 
+    /**
+     * Initialise the image preset on the settings array. This is a custom configuration option that asset-admin reads
+     * to provide some preset image sizes.
+     * @param array $settings
+     */
+    private function initImageSizePresets(array &$settings): void
+    {
+        if (empty($settings['image_size_presets'])) {
+            $settings['image_size_presets'] = self::config()->get('image_size_presets');
+        }
+
+        foreach ($settings['image_size_presets'] as &$preset) {
+            if (isset($preset['i18n'])) {
+                $preset['text'] = i18n::_t(
```

Comment:
Do we need `i18nEntityProvider` here? My guess is textcollector won't find these

Reply:
Yeah, not a guess. Just checked it - collector doesn't see these items.

Reply:
`TinyMCEConfig` now implements `i18nEntityProvider`.